packages/nvlsm/nvlsm.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Start NVIDIA Switch Manager
+RequiresMountsFor=/etc
+
+[Service]
+Type=oneshot
```

Comment:
Is this really a one-shot?

Reply:
This is probably just an `exec` rather than a `oneshot`. If we avoid using `-B` for daemonizing, I think that would give us a `simple` instead but I'd have to experiment with this to see if that is the behavior we really want.